electron/index.d.ts

```diff
@@ -1864,12 +1864,7 @@ declare namespace Electron {
 		/**
 		 * Writes data to the clipboard.
 		 */
-		write(data: {
-			text?: string;
-			rtf?: string;
-			html?: string;
-			image?: NativeImage;
-		}, type?: ClipboardType): void;
+		write(data: { text: string; bookmark?: string; } | { rtf: string; } | { html: string; } | { image: NativeImage; }, type?: ClipboardType): void;
```

Comment:
Well, it appears everything in JavaScript is an object. Funny enough, you can even pass the function itself as an argument and that will still compile: `clipboard.write(clipboard.write)`.